packages/cli/lib/prompts/projectDevTargetAccountPrompt.js

```diff
@@ -28,18 +39,48 @@ const selectTargetAccountPrompt = async (
         };
       });
   } else {
+    let sandboxUsage = {};
+    try {
+      const accountId = getAccountId(defaultAccountConfig.portalId);
+      sandboxUsage = await getSandboxUsageLimits(accountId);
+    } catch (err) {
+      logger.debug('Unable to fetch sandbox usage limits: ', err);
+    }
+    const sandboxAccounts = accounts.reverse().filter(isSandbox);
```

Comment:
I've also reversed the accounts order out of the config to try and put the most recent accounts first. I figured since we don't have an accurate date to pull from, we can assume that new accounts are always added to the bottom of the config

Reply:
Seems reasonable to me 👌